features/task_validate_image.feature

```diff
@@ -281,44 +281,6 @@ Feature: Verify Enterprise Contract Tekton Tasks
       And the task logs for step "report" should match the snapshot
       And the task results should match the snapshot
 
-  Scenario: Effective-time is honored
-    Given a working namespace
-      And a key pair named "known"
-      And an image named "acceptance/effective-time"
-      And a valid image signature of "acceptance/effective-time" image signed by the "known" key
-      And a valid attestation of "acceptance/effective-time" signed by the "known" key
-      And a cluster policy with content:
-      ```
-      {"publicKey": ${known_PUBLIC_KEY}}
-      ```
-    When version 0.1 of the task named "verify-enterprise-contract" is run with parameters:
-      | IMAGES               | {"components": [{"containerImage": "${REGISTRY}/acceptance/effective-time"}]} |
-      | POLICY_CONFIGURATION | ${NAMESPACE}/${POLICY_NAME}                                                   |
-      | IGNORE_REKOR         | true                                                                          |
-      | EFFECTIVE_TIME       | 2020-01-01T00:00:00Z                                                          |
-    Then the task should succeed
-      And the task logs for step "debug-log" should contain "Using provided effective time 2020-01-01T00:00:00Z"
-
-  # Previously we did allow a custom timeout to be set via the TIMEOUT param, but now it's ignored.
-  # (This test could be removed in the future, but let's keep it for now I guess.)
-  Scenario: Deprecated timeout param is ignored
-    Given a working namespace
-      And a key pair named "known"
-      And an image named "acceptance/timeout"
-      And a valid image signature of "acceptance/timeout" image signed by the "known" key
-      And a valid attestation of "acceptance/timeout" signed by the "known" key
-      And a cluster policy with content:
-      ```
-      {"publicKey": ${known_PUBLIC_KEY}}
-      ```
-    When version 0.1 of the task named "verify-enterprise-contract" is run with parameters:
-      | IMAGES               | {"components": [{"containerImage": "${REGISTRY}/acceptance/timeout"}]} |
-      | POLICY_CONFIGURATION | ${NAMESPACE}/${POLICY_NAME}                                            |
-      | IGNORE_REKOR         | true                                                                   |
-      | TIMEOUT              | 666s                                                                   |
-    Then the task should succeed
-      And the task logs for step "debug-log" should contain "globalTimeout is 100h0m0s"
-
```

Comment:
It's too bad we're removing this test coverage. I wonder if there's another way to check the globalTimeout param is working as expected.

Reply:
Since globalTimeout is deprecated, we will remove the check of globalTimeout and check effectiveTime from report-json

Reply:
check the effective-time from report-json.

Reply:
Just to be clear, we're not deprecating the timeout of the CLI. We're only deprecating the timeout of the Tekton Task.

If someone is running the EC CLI directly (locally, for example), the timeout is still useful.